lesson2/local time.py

```diff
@@ -0,0 +1,3 @@
+if __name__ =='__main__':
+    import time
```

Comment:
Define import at the beginning of the file.

Reply:
Done